Gemfile

```diff
@@ -60,7 +60,7 @@ group :development, :test, :staging do
 end
 
 group :development, :test do
-  gem 'puma'
+  gem 'puma', '< 6.0'
```

Comment:
Why?

Reply:
https://github.com/teamcapybara/capybara/pull/2530
https://github.com/puma/puma/blob/master/6.0-Upgrade.md#upgrade
Puma 6.0 break capybara gem. Teamcapybara has fixed in master, but didn't release it yet. There is no critical issues in Puma 5 so I think we could use it until next capybara release.

Reply:
So this is only a test gem. Why would you force it to break an upgrade to a production gem?